git_machete/cmd.py

```diff
@@ -3108,7 +3185,7 @@ def usage(c: str = None) -> None:
               <b>-f, --fork-point=<fork-point-commit></b>    Specifies the alternative fork point commit after which the squashed part of history is meant to start.
         """,
         "status": """
-            <b>Usage: git machete s[tatus] [--color=WHEN] [-l|--list-commits] [-L|--list-commits-with-hashes]</b>
+            <b>Usage: git machete s[tatus] [--color=WHEN] [-l|--list-commits] [-L|--list-commits-with-hashes] [--detect-squash]</b>
```

Comment:
Hmm actually a high-level question: do you think it'd make sense to just make this detect-squash mode the default? What could be the downside of **always** running squash detection? I can think of performance issues since extra calls to `git diff-tree` will happen...

Reply:
This is actually significantly more useful for me if "default detecting" rebase/squash merges.
I've done some very quick benchmarking, and `diff-tree --quiet` is extremely fast.
As per comment below, it's not actually diffing the files directly.